tests/common.sh

```diff
@@ -46,7 +60,7 @@ function exec_check {
 function diff_produced {
     if test -f "$f.expected.out"; then
         if $DIFF -au --strip-trailing-cr -I "executing external script" "$f.expected.out" "$f.produced.out"; then
-            exit 0
+            :
```

Comment:
Just to double-check, is this intentional?

Reply:
I am confused. This change is still present.

Reply:
Yes, I am sorry. The `:` is necessary, and ensures that the rest of the test cases are run ([`:` is a no-op in shell](https://stackoverflow.com/questions/3224878/what-is-the-purpose-of-the-colon-gnu-bash-builtin)).